src/openrct2/world/Entrance.cpp

```diff
@@ -56,8 +55,7 @@ void park_entrance_remove_ghost()
     if (gParkEntranceGhostExists)
     {
         gParkEntranceGhostExists = false;
-        auto parkEntranceRemoveAction = ParkEntranceRemoveAction(
-            { gParkEntranceGhostPosition.x, gParkEntranceGhostPosition.y, gParkEntranceGhostPosition.z * 16 });
```

Comment:
I agree to refactoring out this oddball coordinate, but it will need to be tested thoroughly.